react-delegationdashboard/src/pages/Dashboard/PendingUndelegated/index.tsx

```diff
@@ -16,14 +16,34 @@ const UndelegatedListView = () => {
     return denominate({ denomination, decimals, input: value, showLastNonZeroDecimal: false });
   };
 
+  const getTimeLeft = (value: QueryResponse, index: number, timeLeft: number) => {
+    let roundCurrentEpoch = networkConfig.roundsPerEpoch - networkConfig.roundsPassedInCurrentEpoch;
```

Comment:
optional: can you rename variable to `roundsRemainingInEpoch`?

Reply:
done